Yoast/wordpress-seo
# Update  to use yoastCS 0.4.x

## Summary

This PR can be summarized in the following changelog entry:
_N/A_

## Relevant technical choices:

Raise the YoastCS requirement to 0.4.x

Prerequisites before this PR can be merged:
- [x] Merge PR #6852 up to PR #6873 
- [x] Merge Yoast CS PRs https://github.com/Yoast/yoastcs/pull/30, https://github.com/Yoast/yoastcs/pull/31, https://github.com/Yoast/yoastcs/pull/32
- [x] Tag new Yoast CS release as 0.4.3
- [x] Fix the last five remaining issues or exclude those lines from code style inspection or even exclude the relevant files for those specific issues (see list below)
     Those are issues which I believe someone else should have a look at to decide what should be done.
- [x] Generate new composer.lock file and add it to this branch
- [ ] Rebase this PR after all the merges & fixes are done & hopefully see the build pass ;-)
- [ ] Merge baby, Merge !

## Test instructions

This PR can be tested by following these steps:
* Check if the build passes (requires PR #6874 or an alternative solution to the PHPUnit issue to be merged)

------

```
FILE: wordpress-seo\admin\class-admin-init.php
-----------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-----------------------------------------------------------------------------------
 188 | ERROR | Missing $domain arg. (WordPress.WP.I18n.MissingArgDomain)
-----------------------------------------------------------------------------------

FILE: wordpress-seo\admin\config-ui\class-configuration-page.php
-----------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-----------------------------------------------------------------------------------------------------------
 109 | ERROR | [x] Line indented incorrectly; expected at least 4 tabs, found 4
     |       |     (Generic.WhiteSpace.ScopeIndent.Incorrect)
-----------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-----------------------------------------------------------------------------------------------------------

FILE: wordpress-seo\admin\pages\licenses.php
-----------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-----------------------------------------------------------------------------------------------------------
 217 | ERROR | [x] Line indented incorrectly; expected at least 29 tabs, found 7
     |       |     (Generic.WhiteSpace.ScopeIndent.Incorrect)
-----------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-----------------------------------------------------------------------------------------------------------

FILE: \wordpress-seo\frontend\class-breadcrumbs.php
--------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------------------
 308 | ERROR | Empty IF statement detected (Generic.CodeAnalysis.EmptyStatement.DetectedIF)
--------------------------------------------------------------------------------------------


FILE: \wordpress-seo\frontend\class-opengraph.php
--------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------------------
 805 | ERROR | Empty IF statement detected (Generic.CodeAnalysis.EmptyStatement.DetectedIF)
--------------------------------------------------------------------------------------------
```
